lib/types/enums.ts

```diff
@@ -47,6 +47,9 @@ export enum ReputationEvent {
   PacketTimeout = 2,
   SwapFailure = 3,
   SwapSuccess = 4,
+  UnparseableMessage = 5,
+  InvalidMessage = 6,
+  UnknownMessageType = 7,
```

Comment:
One last thing here, I think this should be called `UnknownPacketType` which is more descriptive and matches the name of the parser error.